File: src/Generated20260410.cpp
#include <iostream>
#include <string>
using namespace std;

class Generated20260410 {
public:
    // Generated class 20260410_164103_382867
    int value = 18;
    string created = "20260410_164103_382867";
    
    int process() {
        int result = value;
        for (int i = 0; i < 6; i++) {
            result += i * 2;
        }
        return result;
    }
};

int main() {
    Generated20260410 obj;
    cout << obj.process() << endl;
    return 0;
}

File: src/Generated20260410.go
package main

import "fmt"

type Generated20260410 struct {
    // Generated struct 20260410_164325_749533
    Value int
    Created string
}

func (c *Generated20260410) Process() int {
    result := c.Value
    for i := 0; i < 13; i++ {
        result += i * 7
    }
    return result
}

func main() {
    obj := Generated20260410{Value: 274, Created: "20260410_164325_749533"}
    fmt.Println(obj.Process())
}